source/docs/api-reference/simulation/simulation-intro.rst

```diff
@@ -126,3 +128,28 @@ Some device ``SimState`` objects also contain outputs that can be used in simula
          // apply the new rotor position and velocity to the TalonFX
          m_talonFXSim.SetRawRotorPosition(m_motorSimModel.GetPosition());
          m_talonFXSim.SetRotorVelocity(m_motorSimModel.GetVelocity());
+
+Feeding Enable (non-FRC)
```

Comment:
We already moved this section to a separate enabling actuators page, so we don't need it here. Non-FRC users already know they need to feedEnable.